kong/plugins/rate-limiting/schema.lua

```diff
@@ -53,6 +53,9 @@ return {
           { redis_host = typedefs.host },
           { redis_port = typedefs.port({ default = 6379 }), },
           { redis_password = { type = "string", len_min = 0 }, },
+          { redis_ssl = { type = "boolean", default = false }, },
+          { redis_ssl_verify = { type = "boolean", default = false }, },
+          { redis_server_name = { type = "string" }, },
```

Comment:
Let's add an entity check (see `entity_checks` below) ensuring that all of these are specified if either one of them is set. This can be accomplished by the `mutually_required` entity check; for example:
`{ mutually_required = { "redis_ssl", "redis_ssl_verify", "redis_server_name" } }`.